images/router/haproxy/conf/haproxy-config.template

```diff
@@ -11,7 +11,7 @@ global
 
   daemon
 {{- with (env "ROUTER_SYSLOG_ADDRESS" "") }}
-  log {{.}} {{env "ROUTER_LOG_FACILITY" "local1"}} {{env "ROUTER_LOG_LEVEL" "warning"}}
+  log {{.}} len {{env "ROUTER_MAX_MSG_LEN" "2048"}} {{env "ROUTER_LOG_FACILITY" "local1"}} {{env "ROUTER_LOG_LEVEL" "warning"}}
```

Comment:
Sorry, upon reflection, can we rename this to:

  ROUTER_LOG_MAX_LEN
or
  ROUTER_LOG_LEN

I want too keep everything for logging grouped with the ROUTER_LOG prefix